.pipelines/cherrypick/CherryPick.yml

```
@@ -0,0 +1,34 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+name: Cherry Pick fast-track commits from PR
+
+trigger:
+- fasttrack/*
+pr: none
+
+variables:
+  - group: "Agent pools (DEV)"
+  - name: agentPool
+    value: "$(DEV_AMD64_Ubuntu_Managed)"
+
+resources:
+  repositories:
+    - repository: CBL-Mariner-Pipelines
+      type: git
+      name: mariner/CBL-Mariner-Pipelines
+
+jobs:
+  - job: 'CherryPick'
+    pool:
+      type: linux
+      isCustom: true
+      name: "$(agentPool)"
```

No need for `agentPool`:
```suggestion

resources:
  repositories:
    - repository: CBL-Mariner-Pipelines
      type: git
      name: mariner/CBL-Mariner-Pipelines

jobs:
  - job: 'CherryPick'
    pool:
      type: linux
      isCustom: true
      name: "$(DEV_AMD64_Ubuntu_Managed)"
```